src/app/content/highlights/components/Highlights.spec.tsx

```diff
@@ -399,6 +399,6 @@ describe('Highlights', () => {
       pageId,
     }));
 
-    expect(component.root.findAllByType(HighlightDeleteWrapper).length).toEqual(1);
+    expect(component.root.findAllByType(HighlightDeleteWrapper).length).toEqual(0);
```

Comment:
This was wrong, right? One highlight gets deleted, other one cancelled so there shouldn't be any more left.

Reply:
it seems like yes
since `state` does not have currentPage highlights clicking delete will dispatch an action but will not close the menu - this is there was one still open